### PR TITLE
Gfs2

### PR DESCRIPTION
**Pull request by lvarin**

### CODE REVIEW

**Comment by trispera** [comment]
Tested both and worked. Need to find a way for creating the `gfs2` filesystem through `ansible`.
I had an error at the beginning of the gfs2 deployment. I was missing the `community.general` ansible collection to use `community.general.dnf_config_manager` task.
It should be added in `requirements.yml`

File: main.yml [hunk 1]
```diff
@@ -6,19 +6,25 @@
     - name: key_name
       prompt: Key name?
       private: false
+      default: alvaro
```
Comment (trispera) [inline main.yml]: Maybe remove this default value?

File: main.yml [hunk 2]
```diff
@@ -6,19 +6,25 @@
...
     - name: network
       prompt: Network?
       private: false
+      default: project_2001316
```
Comment (trispera) [inline main.yml]: Maybe remove this default value?

File: main.yml [hunk 3]
```diff
@@ -6,19 +6,25 @@
...
     - name: jumpHostIP
       prompt: Jump Host IP?
+      default: 86.50.169.241
```
Comment (trispera) [inline main.yml]: Maybe remove this default value?

File: main.yml (hunk 1 above)
Comment (lvarin) [reply main.yml]:
```suggestion
      # default: <username> 
```

File: main.yml (hunk 3 above)
Comment (lvarin) [reply main.yml]:
```suggestion
      # default: <IP>
```

File: main.yml (hunk 2 above)
Comment (lvarin) [reply main.yml]:
```suggestion
      # default: <project_XXXXXXX>
```

**Comment by lvarin** [comment]
> Tested both and worked. Need to find a way for creating the `gfs2` filesystem through `ansible`. I had an error at the beginning of the gfs2 deployment. I was missing the `community.general` ansible collection to use `community.general.dnf_config_manager` task. It should be added in `requirements.yml`

fixed dependencies. Will look into the `mkfs`

**Pull request merged**